thunderfish/lizpulses.py

```diff
@@ -22,12 +22,18 @@
 from .eventdetection import detect_peaks
 from .pulse_tracker_helper import makeeventlist, discardnearbyevents, discard_connecting_eods
 
+import warnings
+
+def warn(*args,**kwargs):
+    pass
+warnings.warn=warn
+
 # upgrade numpy functions for backwards compatibility:
 if not hasattr(np, 'isin'):
     np.isin = np.in1d
 
 
-def extract_pulsefish(data, samplerate, peakwidth=0.002, cutwidth=0.001, **cluster_kwargs):
+def extract_pulsefish(data, samplerate, peakwidth=0.002, cutwidth=0.001, threshold_factor=2, verbose=False, **cluster_kwargs):

```

Comment:
verbose is an integer, because of the multilevel verbosity. Thus it should default to 0:
```
verbose=0
```